futures-core/src/stream/mod.rs

```diff
@@ -34,7 +34,10 @@ pub trait Stream {
     /// Attempt to pull out the next value of this stream, returning `None` if
     /// the stream is finished.
     ///
-    /// This method, like [`Future::poll`](::future::Future::poll),
+    /// This method, like [`Future::poll`](::future::Future::poll), makes
```

Comment:
This sentence feels a bit awkward to me. The docs for `Future::poll` say "Attempt to resolve the future to a final value, registering the current task for wakeup if the value is not yet available." Perhaps we could do something similar here:

"Attempt to resolve the next value in the stream, registering the current task for a wakeup if the value is not yet available."